gpio/README.md

```diff
@@ -390,13 +378,12 @@ A recieved command will be sent back on all configured connections to the config
 
 ### Dependencies
 
-The user running sensor_reporter must have permission to access the GPIO pins.
-To grant the `sensorReporter` user GPIO permissions add the user to the group `gpio`:  `$ sudo adduser sensorReporter gpio`
-
-Depends on RPi.GPIO.
+Depends on `RPi.GPIO` Python library.
+The user running sensor_reporter must be in the `gpio` group to have GPIO access.
 
 ```bash
-sudo pip3 install RPI.GPIO
+cd /srv/sensorReporter
+sudo ./install_dependencies.sh gpio
 ```
 
```

Comment:
We should probably offer an "uninstall" option. At a minimum it should remove the .system file and logs folder.

Reply:
> Would that satisfy your request?

Yes, that would work for me.

> Should we mention this somewhere in the readme?
Absolutely the uninstall option for install_dependencies.sh should be mentioned. That's a good option to make clear is there. 

For me at least it's not uncommon for me to move stuff from one RPi to another which means changing the sensorReporter configs.

Reply:
I'll update this PR with the changes.
At the moment I don't have the time. This should change by the end of February.

Reply:
done